src/Sarif.Viewer.VisualStudio/FixSuggestedAction.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Imaging.Interop;
+using Microsoft.VisualStudio.Language.Intellisense;
+
+namespace Microsoft.Sarif.Viewer
+{
+    class FixSuggestedAction : ISuggestedAction
```

Comment:
>ISuggestedAction [](start = 31, length = 16)

Does this actually derive from IDisposable, or are we adding it?

Reply:
`ISuggestedAction` extends `IDisposable`.

---
In reply to: [509475047](https://github.com/microsoft/sarif-visualstudio-extension/pull/228#discussion_r509475047) [](ancestors = 509475047)